sass/icons/chart settings/chart types/baseline/ic-baseline-normal.svg

```diff
@@ -1,12 +1,12 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
     <g fill="none" fill-rule="evenodd">
```

Comment:
I find it odd you need to use a class `no-stroke`. In `<g />` you can set `stroke="none"  fill="none"` right?